dotcom-rendering/src/web/layouts/InteractiveImmersiveLayout.tsx

```diff
@@ -406,19 +405,9 @@ export const InteractiveImmersiveLayout = ({
 										ArticleSpecial.Labs ? (
 											<GuardianLabsLines />
 										) : (
-											<Lines
-												cssOverrides={css`
-													display: block;
-												`}
-												effect={decideLineEffect(
-													ArticleDesign.Standard,
-													format.theme,
-												)}
-												count={decideLineCount(
-													ArticleDesign.Standard,
-												)}
-											/>
+											<DecideLines format={format} />
 										)}
+										ss
```

Comment:
🤫

Reply:
On its way out ... my `cmd` key has decided that today it no longer wants to work ...